tests/unit_tests/c_wrappers/res/enkf/test_subst_config.py

```diff
@@ -23,6 +23,7 @@ def fixture_snake_oil_structure_config(copy_case):
     cwd = os.getcwd()
     config_file_name = "config"
     return {
+        ConfigKeys.NUM_REALIZATIONS: 1,
```

Comment:
To be certain, this was not supposed to be CPU's?

Reply:
No, this looks correct, due to:
```
 config.write("NUM_REALIZATIONS  1\n")
```

Reply:
yeah that one still needs to be set...